README.md

```diff
@@ -28,6 +28,8 @@ find the SDK.
 $ cordova platform add android
 ```
 
+**Note:** You need to have Android build tools 29.0.2 installed *and not have installed any version greater than 29.0.2*. Due to a [bug](https://github.com/apache/cordova-android/issues/1290) in `cordova-anroid` < 10.0, which this project uses, it's not possible to specify the build tools version with `--gradleArg=-PcdvBuildToolsVersion=29.0.2`. Because of this, it will always try to use the latest installed and fail with "Build-tool is missing DX" if they are greater than 29.0.2.
```

Comment:
Please wrap the line similar to the other lines in here to avoid overly long lines :)